cmd/tracee/cmd/root.go

```diff
@@ -186,34 +186,18 @@ func initCmd() error {
 
 	// Buffer flags
 
-	defaultBufferPages := (4096 * 1024) / os.Getpagesize() // 4 MB of contiguous pages
-	rootCmd.Flags().IntP(
-		"perf-buffer-size",
+	rootCmd.Flags().StringArrayP(
+		flags.BuffersFlag,
 		"b",
-		defaultBufferPages,
-		"<size>\t\t\t\tSize, in pages, of the internal perf ring buffer used to submit events from the kernel",
-	)
-	err = viper.BindPFlag("perf-buffer-size", rootCmd.Flags().Lookup("perf-buffer-size"))
-	if err != nil {
-		return errfmt.WrapError(err)
-	}
-
-	rootCmd.Flags().Int(
-		"blob-perf-buffer-size",
-		defaultBufferPages,
-		"<size>\t\t\t\tSize, in pages, of the internal perf ring buffer used to send blobs from the kernel",
-	)
-	err = viper.BindPFlag("blob-perf-buffer-size", rootCmd.Flags().Lookup("blob-perf-buffer-size"))
-	if err != nil {
-		return errfmt.WrapError(err)
-	}
-
-	rootCmd.Flags().Int(
-		"pipeline-channel-size",
-		1000,
-		"<size>\t\t\t\tSize, in event objects, of each pipeline stage's output channel",
+		[]string{
+			fmt.Sprintf("kernel.events=%d", flags.GetDefaultPerfBufferSize()),
+			fmt.Sprintf("kernel.control-plane=%d", flags.GetDefaultPerfBufferSize()),
+			fmt.Sprintf("kernel.blobs=%d", flags.GetDefaultPerfBufferSize()),
+			"pipeline=10000",
```

Comment:
The current default pipeline channel size is 1000, why changing it to 10000?
This consumes much more memory.